README.md

```diff
@@ -468,6 +469,252 @@ ANSWERS.addComponent('DirectAnswer', {
 })
 ```
 
+## Custom Direct Answer Card
+
+You can customize the look and behavior of your DirectAnswer, you can create a custom Direct Answer card.
```

Comment:
".. of your DirectAnswer by creating a custom Direct Answer card"

Reply:
sorry